packages/esm-patient-chart-app/src/mark-patient-deceased/deceased-form.workspace.tsx

```diff
@@ -0,0 +1,238 @@
+      <div>
+        {isTablet && (
+          <Row className={styles.headerGridRow}>
+            <ExtensionSlot className={styles.dataGridRow} name="visit-form-header-slot" state={memoizedPatientUuid} />
+          </Row>
+        )}
+        <div className={styles.container}>
+          <span className={styles.warningContainer}>
+            <WarningFilled aria-label={t('warning', 'Warning')} className={styles.warningIcon} size={20} />
+            <span className={styles.warningText}>
+              {t('markDeceasedWarning', 'Marking the patient as deceased will end any active visits for this patient')}
+            </span>
+          </span>
+          <section>
+            <div className={styles.sectionTitle}>{t('dateOfDeath', 'Date of death')}</div>
+            {causesOfDeath?.length ? (
+              <ResponsiveWrapper>
+                <Controller
+                  name="deathDate"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <DatePicker
+                      className={styles.datePicker}
+                      dateFormat="d/m/Y"
+                      datePickerType="single"
+                      id="deceasedDate"
+                      maxDate={new Date().toISOString()}
+                      onChange={([date]) => onChange(date)}
+                      value={value}
+                    >
+                      <DatePickerInput
+                        disabled={isDead}
+                        id="deceasedDateInput"
+                        labelText={t('date', 'Date')}
+                        placeholder="dd/mm/yyyy"
+                        style={{ width: '100%' }}
+                      />
+                    </DatePicker>
+                  )}
+                />
+                {errors?.deathDate && <p className={styles.errorMessage}>{errors?.deathDate?.message}</p>}
+              </ResponsiveWrapper>
+            ) : (
+              <DatePickerSkeleton />
+            )}
+          </section>
+          <section>
+            {!isDead && <div className={styles.sectionTitle}>{t('causeOfDeath', 'Cause of death')}</div>}
+
+            <div
+              className={classNames(styles.conceptAnswerOverviewWrapper, {
+                [styles.conceptAnswerOverviewWrapperTablet]: isTablet,
+                [styles.conceptAnswerOverviewWrapperDesktop]: !isTablet,
+                [styles.errorOutline]: errors?.causeOfDeath?.message,
+              })}
+            >
+              {isLoadingCausesOfDeath ? <StructuredListSkeleton /> : null}
+              {causesOfDeath?.length > 10 ? (
+                <ResponsiveWrapper>
+                  <Search
+                    onChange={(event) => setSearchTerm(event.target.value)}
+                    placeholder={t('searchForCauseOfDeath', 'Search for a cause of death')}
+                    labelText=""
+                  />
+                </ResponsiveWrapper>
+              ) : null}
```

Comment:
This change is very arbitrary. On dev3, we currently have 6 cause-of-death concept answers configured. In practice, would we expect implementations to configure a lot more through this form? Or would most implementers elect to use a custom form for Marking patients as deceased? At AMPATH, we used a custom clinical form for this purpose, so I imagine most will be doing something similar. Should I default to showing the Search input anyway, regardless of the number of options in the list?

Reply:
We probably should just show the search input. Yeah, I don't necessarily expect implementations to use this feature as there is very likely to be other perimortem information they need to capture.